### DISCUSSION TITLE
Remove function for Regions

### DISCUSSION THREAD

**Discussion by ManimMadhav** [katspaugh/wavesurfer.js#3623] [Q&A]
What is the status of the ```wsRegions.remove()``` method? I can't seem to find it in the docs on the [Regions](https://wavesurfer.xyz/docs/classes/plugins_regions.default.html) or on the [Methods](https://wavesurfer.xyz/docs/classes/wavesurfer.default) page either. 
I am using WaveSurfer.js from here: 'https://unpkg.com/wavesurfer.js@7/dist/wavesurfer.esm.js' .
Is ```region.remove()``` obsolete now? It seems to be completing the functionality of deleting a region but everytime I use it it throws a warning, as shown below:<br>
![image](https://github.com/katspaugh/wavesurfer.js/assets/66988314/b6df9bfc-7c7a-4dfb-8e58-24e3b899aa12)

**Comment by katspaugh** [comment]
You can remove an individual region via ~`region.destroy()`~ `region.remove()`. Docs: https://wavesurfer.xyz/docs/modules/plugins_regions

**Reply by ManimMadhav** [reply]
wait so how has ```region.remove()``` been working for me all this time?

**Reply by katspaugh** [reply]
Ah yes, sorry, you're right. It's `region.remove()`. There's no `wsRegions.remove()` though.

**Reply by ManimMadhav** [reply]
Oh yes, so basically it's ```wsRegions.destroy()``` to remove the plugin itself, and ```region.remove()``` to delete a region.
So why is ```region.remove()``` not available in documentation?

**Reply by katspaugh** [reply]
Yeah, that's unfortunate that the docs for the Region class aren't displayed at all. Something to do with how typedoc treats exports. Can be probably fixed by using a type alias for the Region class and exporting both separately.
